lib/calabash/http/retriable_client.rb

```diff
@@ -25,7 +25,7 @@ def get(request, options={})
             @client.get(@server.endpoint + request.route, request.params)
           end
         rescue => e
-          raise HTTP::Error.new(e.message)
+          raise Calabash::HTTP::Error.new(e.message)
```

Comment:
No.  HTTP::Error != Calabash::HTTP::Error

And more importantly, to use HTTP::Error in my rspec tests, I need to require 'httpclient'.